client/components/tinymce/style.scss

```diff
@@ -1,9 +1,9 @@
-@import 'plugins/wplink/style';
-@import 'plugins/calypso-alert/style';
-@import 'plugins/insert-menu/style';
-@import 'plugins/wpcom-help/style';
-@import 'plugins/wpcom-charmap/style';
-@import 'plugins/contact-form/style';
+@import 'plugins/wplink/wplink';
```

Comment:
@aduth / @mtias — how do you feel about these changes? Rationale here: https://github.com/Automattic/wp-calypso/pull/9270#issuecomment-260674924